hack/make-docker-images.sh

```diff
@@ -38,9 +38,11 @@ while IFS= read -d $'\0' -r dir; do
         cd "${builddir}"
         log "Building: ${image}"
         docker build -t "${image}" .
+        docker build -t "${image}-native-grpc-probes" . --target without-grpc-health-probe-bin
 
         log "Pushing: ${image}"
         docker push "${image}"
+        docker push "${image}-native-grpc-probes"
```

Comment:
Praise: Thanks for building and pushing the new images in the release scripts (`/hack/make-release.sh`).